.teamcity/_self/projects/WebApp.kt

```diff
@@ -8,6 +8,7 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.*
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.dockerCommand
 import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.*
 import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.schedule
```

Comment:
I think this line can be removed now :) (maybe moved to the WPComTests file?)

Reply:
good catch, thanks!